README.md

```diff
@@ -1,70 +1,47 @@
-# Getting Started with Create React App
+# City-Weather App
```

Comment:
- [ ] [REQUIRED] Kindly ensure you make a video presentation of this project with a 5-minute maximum length. Also, ensure the video has a clear presentation of the project with a highlight of a piece of code you are most proud of. This is a core requirement.